docs/UserGuide.md

```diff
@@ -154,76 +281,92 @@ Format: `clear`
-* For email, country, phone and interview status, only full words will be matched.
+* For email, country and phone, only full words will be matched.
 * For name, comment and tag, partial words will be matched, e.g. `Han` will match `Hans`.
-* The search will fail if either of the email, country, phone or interview status is in an invalid format.
+* The search will fail if either of the email, country, phone is in an invalid format.
 * If multiple fields are specified, only candidates that match **all** the specified attributes will be returned.
 
 Examples:
-* `search n/John s/ACCEPTED` returns candidates whose names contain `John` and whose interview status is `ACCEPTED`.
+* `search n/John c/SG` returns candidates whose names contain `John` and whose country is from `Singapore`.
 * `search t/Internal` returns candidates whose tags contain `Internal`.
 
-### Accessing by index: `get`
+### Search for matching jobs : `search_job`
 
-Access candidates by index
-* `INDEX` must be within the range `1` to `n`, where `n` is the number of records in the database.
+Searches jobs whose attributes match the specified attributes in the search criteria.
 
-Format: `get INDEX`
+Format: `search_job [ti/TITLE] [d/DESCRIPTION] [v/VACANCY]`
 
-Example:
-* `get 24` returns the candidate with index 24
+* At least one of the optional fields must be provided.
+* The search is case-sensitive, e.g. `engineer` will not match `Engineer`.
+* For title and description, partial words will be matched.
+* For vacancy, only jobs with the same number of vacancies will be matched.
+* If multiple fields are specified, only jobs that match **all** the specified attributes will be returned.
 
-### Adding a comment on a candidate: `comment`
+Examples:
+* `search_job ti/Engineer d/C++` returns jobs whose titles contain `Engineer` and whose description contain `C++`.
+* `search_job v/10` returns jobs whose number of vacancies is 10.
 
-Leaves comments on important points to note down for individual candidates during the recruitment process. This overwrites existing comment (if any) and displays the resulting candidate.
+### Search for matching applications : `search_app`
 
-* `INDEX` must be within the range `1` to `n`, where `n` is the number of records in the database.
-* Any comment format is acceptable as long as comment is non-empty (i.e. user writes nothing in the comment field)
+Searches applications whose attributes match the specified attributes in the search criteria.
 
-Format: `comment INDEX m/COMMENT`
+Format: `search_app [e/EMAIL] [ti/TITLE] [s/STATUS]`
 
-Example:
-* `comment 3 m/Managed to solve every round 3 interview questions. He must be a strong candidate, potentially to be recruited as a quantitative research intern at Jane Street.`
+* At least one of the optional fields must be provided.
+* The search is case-sensitive, e.g. `engineer` will not match `Engineer`.
+* For title, partial words will be matched.
+* For email and status, only full words will be matched.
```

Comment:
can add that status is case-insensitive

Reply:
I didn't know that, will specify, thanks!